{"uuid": "5d56e2d5-6255-437e-9e48-0dde886c58a5", "children": ["06af33ae-28df-4f67-9acb-c8eccdf8f844"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "KeyError: 'check'\n", "trace": "  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/nali/AutoApiTest/conftest.py\", line 28, in login\n    api_send_check(request.param, case_dict, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/api_send_check.py\", line 29, in api_send_check\n    check_data = case_data['check']\n"}, "steps": [{"name": "POST请求接口", "status": "passed", "attachments": [{"name": "请求接口：", "source": "8e480a71-66d4-4b5c-9101-e67a739ade5f-attachment.attach"}, {"name": "用例描述：", "source": "539eea28-a7cf-4992-86ea-4d14e057c25d-attachment.attach"}, {"name": "请求地址", "source": "30fe178b-ae5d-49d2-a53a-efcded32ef88-attachment.attach"}, {"name": "请求头", "source": "8050d995-71ad-4b16-8a61-041c30cb81ff-attachment.attach"}, {"name": "请求参数", "source": "dc4f2812-0b96-4604-a4a8-286b103caaa7-attachment.attach"}], "start": 1617289439452, "stop": 1617289439454}], "start": 1617289439449, "stop": 1617289439642}], "start": 1617289439449, "stop": 1617289439676}